src/CMakeLists.txt

```diff
@@ -410,6 +410,8 @@ dbms_target_link_libraries(PRIVATE ch_contrib::zstd)
 target_link_libraries (clickhouse_common_io PUBLIC ch_contrib::zstd)
 target_link_libraries (clickhouse_common_io PUBLIC ch_contrib::xz)
 
+target_link_libraries(clickhouse_common_io PUBLIC ch_contrib::pocketfft)
```

Comment:
Needs conditional wrapping:

```cmake
if (TARGET ch_contrib::pocketfft)
    target_link_libraries(clickhouse_common_io PUBLIC ch_contrib::pocketfft)
endif ()
```

(otherwise builds with `-DENABLE_LIBRARIES=0` will break)